kernel/sched.c

```diff
@@ -8,14 +8,18 @@ static const struct sched *sched;
 
 int sched_select(int sched_type, struct thread_info *thread)
 {
+    printk("Scheduler: ");
```

Comment:
We don't need the invocation of `printk` in order to display scheduler internals literally. Instead, we can always deploy custom GDB script to perform deeper and effective tracing over existing framework.

Reply:
Sure, I'll take a reference from f9-kernel, except gdb script, maybe some probe can be plugin.